ContentFiles/WPF/HomeView.xaml.cs.pp

```diff
@@ -0,0 +1,15 @@
+using AlphanumericOz.Core.ViewModels;
```

Comment:
I think this should be `using $rootnamespace$.ViewModels;` 🙂

Reply:
yep, good pickup, accidentally left my sample project in there.